digdag-standards/src/main/java/io/digdag/standards/command/ecs/TemporalProjectArchiveStorage.java

```diff
@@ -55,4 +55,11 @@ public InputStream getContentInputStream(final String key)
             throw ThrowablesUtil.propagate(e);
         }
     }
+
+    public String getBucketName(final Config systemConfig)
```

Comment:
I understand this method expects to get an S3 bucket name so I believe we can simplify this like
```suggestion
    public String getS3BucketName(final Config systemConfig)
    {
        return systemConfig.get(PARAMS_PREFIX + "s3.bucket", String.class);
    }    
```

Reply:
I see. The storage is expected to be S3 so I fixed type with it on this commit https://github.com/treasure-data/digdag/pull/1762/commits/02030bd322a591c0eecfb761622eec65a7ee316b.